### PR TITLE
Add CFScript example for CFOAuth

### PR DESCRIPTION
**Pull request by brilang**

### CODE REVIEW

**Comment by brilang** [comment]
I don't know why this isn't passing the isJSON check in Travis. The code string passes isJSON in CF 2016 for me. I'd appreciate some help figuring out what's going on...

**Comment by brilang** [comment]
Finally. This is ready to be merged assuming there are no other issues.

File: data/en/cflogin.json [hunk 1]
```diff
@@ -16,5 +16,15 @@
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/tag/cflogin"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/tag/cflogin"}
 	},
-	"links": []
+	"links": [],
+  "examples": [
```
Comment (shaedrich) [inline data/en/cflogin.json]: Could you indent it properly?

File: data/en/cfoauth.json [hunk 2]
```diff
@@ -111,7 +111,15 @@
...
+	  "code": "<cflogin>\r\n<cfoauth\r\n\ttype=\"Facebook\"\r\n\tclientid=\"YOUR_CLIENT_ID\"\r\n\tsecretkey=\"YOUR_SECRET_KEY\"\r\n\tresult=\"res\"\r\n\tredirecturi=\"http:\/\/localhost:8500\/doc\/login.cfm\"\r\n>\r\n<cfloginuser name=\"#res.other.username#\" password=\"#res.access_token#\" roles=\"user\"\/>\r\n<\/cflogin>\r\n<cflocation url=\"http:\/\/localhost:8500\/doc\/index.cfm\">",
+	  "runnable":false
+	},
+	{
+	  "result": "",
+	  "title": "CFSCRIPT Example",
+	  "description": "The following example shows a simple authentication:",
+	  "code": "\n cflogin() {\n cfoauth(\n type=\"Google\",\n clientid=\"YOUR_CLIENT_ID\",\n secretkey=\"YOUR_GOOGLE_CLIENTSECRET\",\n redirecturi=\"YOUR_CALLBACK_URI\",\n result=\"res\",\n scope=\"YOUR_SCOPES\",\n state=\"cftoken=#cftoken#\"\n );\n cfloginuser(name=\"#res.other.email#\", password=\"#res.access_token#\", roles=\"user\");\n }\n",
```
Comment (shaedrich) [inline data/en/cfoauth.json]: Could you remove the `\n` at the beginning of the line?

**Pull request merged**